solar_system.rb

```diff
@@ -0,0 +1,92 @@
+#WAVE 3 BASIC
+
+class Planet
+  attr_reader :names, :diameter, :mass, :number_moons, :color
+
+  def initialize(name, diameter, mass, number_moons, color)
+    @name = name
+    @diameter = diameter
+    @mass = mass
+    @number_moons = number_moons
+    @color = color
+  end
+
+  def name
+    return @name
+  end
+
+  def display_planets
+    return "name: #{@name}, diameter: #{@diameter}, mass: #{@mass}, number of moons: #{@number_moons}, color: #{@color} \n"
+  end
+end
+
+class Solar_system
+  attr_reader :planets
+
+  def initialize(planets)
+    @planets = planets
+  end
+
+  def add_planet(planet)
+    @planets << planet
+  end
+
+  def list_planets
+    x = 0
+    @planet_names = ""
+    @planets.each do |planet|
+      x+=1
+      @planet_names << "#{x}. #{planet.name} \n"
+    end
+    return @planet_names
+  end
+
+  def make_planet_array
+    @planet_array = []
+    @planets.each do |planet|
+      @planet_array << "#{planet.display_planets}"
+    end
```

Comment:
Good use of the existing `display_planets` method on the `Planet` class. Makes this code very readable.